src/cascadia/TerminalApp/TerminalPage.h

```diff
@@ -6,12 +6,144 @@
+
+        void _HookupKeyBindings(TerminalApp::AppKeyBindings bindings) noexcept;
+
+        void _UpdateTitle(std::shared_ptr<Tab> tab);
+        void _UpdateTabIcon(std::shared_ptr<Tab> tab);
+        void _UpdateTabView();
+        void _DuplicateTabViewItem();
+        void _RemoveTabViewItem(const IInspectable& tabViewItem);
+
+        void _RegisterTerminalEvents(Microsoft::Terminal::TerminalControl::TermControl term, std::shared_ptr<Tab> hostingTab);
+
+        void _SelectNextTab(const bool bMoveRight);
+        bool _SelectTab(const int tabIndex);
+        void _MoveFocus(const Direction& direction);
+
+        winrt::Microsoft::Terminal::TerminalControl::TermControl _GetFocusedControl();
+        int _GetFocusedTabIndex() const;
+        void _SetFocusedTabIndex(int tabIndex);
+        void _CloseFocusedTab();
+        void _CloseFocusedPane();
+
+        // Todo: add more event implementations here
+        // MSFT:20641986: Add keybindings for New Window
+        void _Scroll(int delta);
+        void _SplitVertical(const std::optional<GUID>& profileGuid);
+        void _SplitHorizontal(const std::optional<GUID>& profileGuid);
+        void _SplitPane(const Pane::SplitState splitType, const std::optional<GUID>& profileGuid);
+        void _ResizePane(const Direction& direction);
+        void _ScrollPage(int delta);
+        static Windows::UI::Xaml::Controls::IconElement _GetIconFromProfile(const ::TerminalApp::Profile& profile);
+        void _SetAcceleratorForMenuItem(Windows::UI::Xaml::Controls::MenuFlyoutItem& menuItem, const winrt::Microsoft::Terminal::Settings::KeyChord& keyChord);
+
+        void _CopyToClipboardHandler(const IInspectable& sender, const winrt::Microsoft::Terminal::TerminalControl::CopyToClipboardEventArgs& copiedData);
+        void _PasteFromClipboardHandler(const IInspectable& sender,
+                                        const Microsoft::Terminal::TerminalControl::PasteFromClipboardEventArgs& eventArgs);
+        bool _CopyText(const bool trimTrailingWhitespace);
+        void _PasteText();
+        static fire_and_forget PasteFromClipboard(winrt::Microsoft::Terminal::TerminalControl::PasteFromClipboardEventArgs eventArgs);
+
+        void _OpenSettings();
+        fire_and_forget LaunchSettings();
+
+        void _OnTabClick(const IInspectable& sender, const Windows::UI::Xaml::Input::PointerRoutedEventArgs& eventArgs);
+        void _OnTabSelectionChanged(const IInspectable& sender, const Windows::UI::Xaml::Controls::SelectionChangedEventArgs& eventArgs);
+        void _OnTabItemsChanged(const IInspectable& sender, const Windows::Foundation::Collections::IVectorChangedEventArgs& eventArgs);
+        void _OnContentSizeChanged(const IInspectable& /*sender*/, Windows::UI::Xaml::SizeChangedEventArgs const& e);
+        void _OnTabClosing(const IInspectable& sender, const Microsoft::UI::Xaml::Controls::TabViewTabClosingEventArgs& eventArgs);
+
+        void RefreshUIForSettingsReload();
```

Comment:
this new private method doesn't begin with a `_` like all the other ones

Reply:
added

Reply:
added